elastio-ec2-launch-template-user-data-examples/README.md

```diff
@@ -0,0 +1,57 @@
+# Create EC2 Launch Template to Automate Elastio Tasks
+
+### Author: Robert Saylor - customphpdesign@gmail.com
+
+---
+
+### Use Cases
+- Install Elastio on Ubuntu
+- Automate backing up DynamoDB
+
+---
+Before you can use the user data in the advanced section when you launch a new EC2 you must first set up the EC2 with a role.
+
+### IAM Role
+
+From IAM click on Roles.
+
+Click on Create role
+
+Select AWS Service and select EC2 from the Common use cases.
+
+Filter the policies you wish to assign to the role. Elastio comes with 6 policies. Assign all the Elastio policies to your role. See [Elastio Policies](https://docs.elastio.com/src/getting-started/elastio-policies.html) for more details.
```

Comment:
Why would the user assign *all* policies to their role?  If they're just doing a stream backup they don't need all of those permissions...

Reply:
I adjusted the text so they will decide what policies to attach.